src/Cake.Common/Tools/SpecFlow/SpecFlowAliases.cs

```diff
@@ -8,7 +8,13 @@
 namespace Cake.Common.Tools.SpecFlow
 {
     /// <summary>
-    /// Contains functionality for working with SpecFlow.
+    /// <para>Contains functionality related to <see href="http://www.specflow.org/">SpecFlow</see>.</para>
+    /// <para>
+    /// In order to use the commands for this alias, include the following in your build.cake file to download and install from NuGet.org, or specify the ToolPath within the Settings class:
```

Comment:
There are `SpecFlowSettings`, `SpecFlowStepDefintionReportSettings` and `SpecFlowTestExectionReportSettings` so was going for a generic mention of a settings class.